### DISCUSSION TITLE
Running on mac terminal

### DISCUSSION THREAD

**Discussion by BananaGeneral** [TermuxHackz/X-osint#18] [Q&A] [answered]
Is it possible to run natively on macos' bash terminal? 
If so, would be nice to have a tutorial on how to do so.

**Comment by TermuxHackz** [comment, marked as answer]
I haven't tested it on a macos bash terminal, however I'm gonna do so and make a tutorial on it.
Thank you

**Comment by BananaGeneral** [comment]
Excellent! Will be looking forward to that. 
Thx

**Comment by TermuxHackz** [comment]
Hi hello, @BananaGeneral , Its fully compatible with the MacOS bash terminal, Sorry it's taken me too long to get back to you , bug fixes have been made and features added!.
